stdlib/3/urllib/request.pyi

```diff
@@ -157,17 +157,25 @@ class ProxyDigestAuthHandler(BaseHandler, AbstractDigestAuthHandler):
     def http_error_407(self, req: Request, fp: IO[str], code: int, msg: int,
                        hdrs: Mapping[str, str]) -> Optional[_UrlopenRet]: ...
 
-class HTTPHandler(BaseHandler):
-    def http_open(self, req: Request) -> HTTPResponse: ...
-    def do_open(self,  # undocumented
+class AbstractHTTPHandler(BaseHandler):  # undocumented
+    def __init__(self, debuglevel: int = ...) -> None: ...
+    def set_http_debuglevel(self, level: int) -> None: ...
+    def do_request_(self, request: Request) -> Request: ...
+    def do_open(self,
                 http_class: HTTPConnectionProtocol,
-                req: Request) -> HTTPResponse: ...
+                req: Request,
+                **http_conn_args: Optional[ssl.SSLContext]) -> HTTPResponse: ...
```

Comment:
The `http_conn_args` comment from the Python 2 version above applies here as well.